include/zephyr/dt-bindings/dma/atmel_samx7x_dma.h

```diff
@@ -37,8 +41,7 @@
 #define DMA_PERID_UART3_RX    27
 #define DMA_PERID_UART4_TX    28
 #define DMA_PERID_UART4_RX    29
-#define DMA_PERID_DACC0_TX    30
-#define DMA_PERID_DACC1_TX    31
+#define DMA_PERID_DACC_TX     30
```

Comment:
The table is wrong. There are 2 channels and original is correct.
https://github.com/zephyrproject-rtos/hal_atmel/blob/d37df06556f7c03ff3a20422475077fafa0dc17c/asf/sam/include/same70b/component/xdmac.h#L582-L583

Reply:
Thank you for reviewing.

Okay, so the `dts/arm/atmel/dma_atmel_same70.h` header had it wrong. I've brought back `DACC0_TX and `DACC1_TX`.